lib/plugins/digging.js

```diff
@@ -83,8 +83,9 @@ function inject (bot) {
   }
 
   function digTime (block) {
+    const blockAtPosition = bot.blockAt(bot.entity.position)
     return block.digTime(bot.heldItem ? bot.heldItem.type : null, bot.game.gameMode === 'creative',
-      bot.blockAt(bot.entity.position).type === 9, !bot.entity.onGround) // only stationary water counts
+      blockAtPosition && blockAtPosition.type === 9, !bot.entity.onGround) // only stationary water counts
```

Comment:
Can you put blockAtPosition !== null here ?
(It's better not to rely on js implicit conversion)

Reply:
Good point